llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md

```diff
@@ -10,10 +10,19 @@
 
 > Knowledge base can be configured through [AWS Console](https://aws.amazon.com/console/) or by using [AWS SDKs](https://aws.amazon.com/developer/tools/).
 
-### Notebook
+## Installation
 
-Explore the retriever using Notebook present at:
+```
+pip install llama-index-retrievers-bedrock
+```
+
+## Usage
 
 ```
-docs/docs/examples/retrievers/bedrock_retriever.ipynb
+from llama_index.retrievers.bedrock import AmazonKnowledgeBasesRetriever
```

Comment:
Maybe useful to show how the obj can be used in addition to being imported?

Reply:
I just copied the example from the notebook lol

Reply:
lol perfect